src/server.ts

```diff
@@ -25,8 +27,39 @@ const authenticationBasePath = container.resolve<string>(
 const APIBasePath = container.resolve<string>("APIBasePath");
 const PagoPABasePath = container.resolve<string>("PagoPABasePath");
 
+// Set default for graceful-shutdown
+const DEFAULT_SHUTDOWN_SIGNALS = "SIGINT SIGTERM";
+const DEFAULT_SHUTDOWN_TIMEOUT_MILLIS = 30000;
+
+const shutdownSignals: string =
+  process.env.SHUTDOWN_SIGNALS || DEFAULT_SHUTDOWN_SIGNALS;
+
+const shutdownTimeout: number = process.env.DEFAULT_SHUTDOWN_TIMEOUT_MILLIS
+  ? parseInt(process.env.DEFAULT_SHUTDOWN_TIMEOUT_MILLIS, 10)
+  : DEFAULT_SHUTDOWN_TIMEOUT_MILLIS;
+
 // tslint:disable-next-line: no-let
 let server: http.Server | https.Server;
+
+export const serverShutdown = (app: Express) => {
+  function cleanup(): Promise<void> {
+    return new Promise(resolve => {
+      log.info("Gracefully shutting down in progress");
+      server.close(() => {
```

Comment:
this is already called by [httpGracefulShutdown](https://github.com/sebhildebrandt/http-graceful-shutdown/blob/master/lib/index.js#L102), why is it also needed here?